src/main/java/frc/robot/Robot.java

```diff
@@ -35,7 +35,7 @@
 	private GrabberFSM grabberSystem;
 
 	// Constants
-	private static final boolean RUN_COMPRESSOR = true;
+	private static final boolean RUN_COMPRESSOR = false;
```

Comment:
Keep this true on main branch. In fact maybe we should put this on shuffleboard.